tensorboard/webapp/widgets/data_table/types.ts

```diff
@@ -43,6 +42,44 @@ export enum ColumnHeaderType {
   CUSTOM = 'CUSTOM',
 }
 
+export enum DomainType {
+  DISCRETE,
+  INTERVAL,
+}
+
+export type DiscreteFilterValues = string[] | number[] | boolean[];
+
+export type DiscreteFilterValue = DiscreteFilterValues[number];
```

Comment:
These two have been copied and renamed rather than moved. Once the old runs table is removed the old versions can be removed.